parastell.py

```diff
@@ -292,80 +294,43 @@ def graveyard(vmec, offset, components, logger):
     return components
 
 
-def offset_point(vmec, s, zeta, theta, offset):
+def offset_point(vmec, s, phi, theta, offset, plane_norm):
     """Stellarator offset surface root-finding problem.
 
     Arguments:
         vmec (object): plasma equilibrium object.
         s (float): normalized magnetic closed flux surface value.
-        zeta (float): toroidal angle being solved for (rad).
+        phi (float): toroidal angle being solved for (rad).
         theta (float): poloidal angle of interest (rad).
         offset (float): total offset of layer from plamsa (m).
+        plane_norm (array of float): normal direction of toroidal plane.
```

Comment:
My thought was to calculate it once per toroidal angle in the `stellarator_torus` method, since it will be the same for each `phi`, rather than compute it repeatedly for each point in `offset_point`